apps/frontend/src/libs/components/input/styles.module.css

```
@@ -0,0 +1,25 @@
+.input-container {
+	display: flex;
+	flex-direction: column;
+}
+
+.input-label {
+	font-size: medium;
+}
+
+.input {
+	padding: 10px;
+	font-size: 16px;
+	border: 2px solid #e0e0e0;
```

You should use color vars 
```suggestion
	border: 2px solid var(--LIGHT_GRAY);

```

---

Yes, thank you.